UnitsNet.Tests/GeneratedCode/AccelerationTestsBase.g.cs

```diff
@@ -327,5 +327,18 @@ public void UnitsDoesNotContainUndefined()
         {
             Assert.DoesNotContain(AccelerationUnit.Undefined, Acceleration.Units);
         }
+
+        [Fact]
+        public void AllUnitsHaveAtLeastOneAbbreviationSpecified()
+        {
+            var units = Enum.GetValues(typeof(AccelerationUnit)).Cast<AccelerationUnit>();
+            foreach(var unit in units)
+            {
+                if(unit == AccelerationUnit.Undefined)
+                    continue;
+
+                var defaultAbbreviation = UnitAbbreviationsCache.Default.GetDefaultAbbreviation(unit);
```

Comment:
Also, it would be helpful to add a comment that we are testing that this does not throw exceptions. It might not be obvious to the reader. Removing the `var defaultAbbreviation` variable would also help highlight this.